pkg/envoy/bootstrap/config.go

```diff
@@ -141,7 +141,7 @@ func BuildFromConfig(config Config) (*xds_bootstrap.Bootstrap, error) {
 			Clusters: []*xds_cluster.Cluster{
 				{
 					Name:           config.XDSClusterName,
-					ConnectTimeout: durationpb.New(time.Millisecond * 250),
+					ConnectTimeout: durationpb.New(time.Second * 3),
```

Comment:
1. Why is this being changed?
2. Why such a high timeout (order of several seconds)?

Reply:
I saw timeouts on Envoy logs while I was testing this. Given that the default timeout is 5s then 0.25s seems a bit low.

Is this reliable on linux clusters? The error is recoverable so you need to see how this value behaves across cluster sizes/loads

Reply:
I agree this is something that depends on traffic patterns and the environment in general, but bumping this timeout to get tests to pass seems a bit mysterious. At the very least, we should set a well-documented constant (even prior to this change) with a comment on why this value.

Reply:
> but bumping this timeout to get tests to pass seems a bit mysterious


The test would pass regardless, it just makes for better logging because it reduces the noise from intermittent failures when reaching the control plane.

> At the very least, we should set a well-documented constant (even prior to this change) with a comment on why this value.

I was thinking that maybe we should not set this value and let Envoy set the default. WDYT? Otherwise I can push it to a constant with a comment

Reply:
@davinci26 if the default value works well, sure that seems reasonable.

Reply:
I just removed it so we let envoy decide on the default